Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM python:3.11-slim
+FROM python:3.13-slim
 
-WORKDIR /code
+WORKDIR /app
 
-COPY ./requirements.txt /code/requirements.txt
+COPY ./requirements.txt /app/requirements.txt
 
-COPY ./config.yaml /code/config.yaml
+COPY ./config.yaml /app/config.yaml
 
-RUN pip install --no-cache-dir -r /code/requirements.txt
+VOLUME /app/data
```

Comment:
I think this should be `VOLUME /data`.

`data` and `app` live alongside each other.

Reply:
Again you are right :). Thanks. I pushed the new commit. I hope that will be okay.